README

```diff
@@ -2,7 +2,7 @@
 
 The program FORKLIFT is an inclusion checker for Büchi automata.
 It takes two automata as input, presented the format described here:
-    http://languageinclusion.org/doku.php?id=tools#the_ba_format
+    https://github.com/parof/bait#the-ba-format
```

Comment:
We also have it on archive.org: https://web.archive.org/web/20230329055152/http://languageinclusion.org/doku.php?id=tools#the_ba_format